### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by porres**
minor updates to the manual and improving release notes ( closes https://github.com/pure-data/pddp/issues/199 )

also @ben-wes ben made updates that
closes https://github.com/pure-data/pddp/issues/200
closes https://github.com/pure-data/pddp/issues/201
closes https://github.com/pure-data/pddp/issues/202
closes https://github.com/pure-data/pddp/issues/203

### CODE REVIEW

File: doc/1.manual/x5.htm [hunk 1]
```diff
@@ -28,14 +28,25 @@ <H3> <A id="s5.1"> 5.1. Release notes </A> </H3>
...
+<P> 64-bit soundfile reading and writing support by Dan Wilcox for [readsf~],
+[writesf~] and [soundfiler].
 
 <P> Many bug fixes and code-level updates and improvements, notably to libpd and
-for emscripten support, thanks as usual to Iohanes, Dan, and Christof.  The
-emscripten support, in particular, should someday allow Pd to run in a web
-browser.
+for emscripten support thanks to Claude Heiland-Allen and IOhannes, Dan, and
+Christof. The emscripten support, in particular, should someday allow Pd to run
+in a web browser.
```
Comment (Spacechild1) [inline doc/1.manual/x5.htm:50]: Maybe you could also mention the new empty `[set(` method for `[throw~]` and `[receive~]`. `sys_getfunbyname` is probably worth mentioning as well.

**Comment by porres** [comment]
oh my god, conflicts, let me take care of this

**Comment by porres** [comment]
hold on a second @millerpuckette I just realized there are conflicts, I guess they came up in the last commit by @ben-wes  let me fix it so you can merge

**Comment by millerpuckette** [comment]
I just bashed through the conflicts.  Meanwhile the new x5.htm wasn't in 
the PR (unless I got confused somehow) and so I just dug it out of an 
older PR and stuffed it into master.

**Comment by ben-wes** [comment]
> I guess they came up in the last commit by @ben-wes let me fix it so you can merge

hmmm ... if you see what i did wrong there, let me know, @porres ! afaict, my changes were based on your `manual-updates` branch.

**Comment by porres** [comment]
@millerpuckette the release note updates are in here at https://github.com/pure-data/pure-data/pull/2292/commits/203e9502f3ea3435f256e8a60cb2cb68bb4e01fa

 I just fixed the conflicts and now we can also close other issues, see if you can take this

**Comment by porres** [comment]
@Spacechild1 I added the [set( method for [throw~] and [receive~] as you requested into the release notes at https://github.com/pure-data/pure-data/pull/2292/commits/8a97e70a66a86705842274dbcf2125845034aecf

but I wouldn't know about how to mention sys_getfunbyname but we can leave it for later, as I'm hoping @millerpuckette iller takes this now for the test release

**Pull request merged**